lib/manager/helmfile/extract.ts

```diff
@@ -43,15 +42,15 @@ export function extractPackageFile(
       if (!is.string(dep.chart)) {
         return {
           depName: dep.name,
-          skipReason: SkipReason.InvalidName,
+          skipReason: 'invalid-name',
         };
       }
 
       // If starts with ./ is for sure a local path
       if (dep.chart.startsWith('./')) {
         return {
           depName,
-          skipReason: SkipReason.LocalChart,
+          skipReason: 'local-chart',
```

Comment:
```suggestion
          skipReason: 'local-dependency',
```

Should we unify this with others?